packages/flame/lib/src/events/messages/drag_update_event.dart

```diff
@@ -36,4 +36,10 @@ class DragUpdateEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'DragUpdateEvent(pointerId: $pointerId, timestamp: $timestamp, '
+      'delta: $delta, _nanPoint: $_nanPoint, devicePosition:$devicePosition, '
+      'canvasPosition:$canvasPosition)';
```

Comment:
`_nanPoint` is useless to the user (it's a constant). 
Similarly to DragStartEvent, let's have `canvasPosition+localPosition` first, followed by `delta`, followed by `pointerId`/`timestamp`.